CONTRIBUTING.md

```diff
@@ -89,6 +89,42 @@ After build completes, you can preview the resulting site locally using built-in
 	
 This will start the local preview on localhost:8080. You can then view the changes by going to http://localhost:8080/docs/[path] (e.g. http://localhost:8080/docs/getting-started/).   
 
-## Contributing to samples
+# Contributing to samples
 
-See the [Samples Readme](https://github.com/dotnet/core-docs/blob/master/samples/README.md).
+We encourage samples for any of our topics. Readers can download and run the samples. They
+can explore the code and experiment with their own changes.
+
+All samples should be complete applications or libraries. Where the sample creates a library,
+it should include unit tests or an application that lets readers run the code.
+
+The samples are organized in directories that match the directories under
```

Comment:
This part could be clearer, mostly around file paths. For example, specify the exact location of `getting-started` to avoid any room for confusion. The `docs/*/tutorials` is too confusing (the star part), IMO, for people new to the repo.